dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/h2/V7.0_2020.01.08__Statistics-harvester.sql

```diff
@@ -0,0 +1,29 @@
+--
```

Comment:
The name of this SQL file is odd to me.  Here we use the term "harvester", but the table is a "tracker" and doesn't seem to be harvesting from anything.  Should this SQL file be renamed?  It also likely should reference to ticket under which it was created.  So, it might be named something like `V7.0_2020.01.08__DS-626-statistics-tracker.sql` or similar.

Reply:
Changed the name of the file